src/App.tsx

```diff
@@ -1,24 +1,206 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo, FilterType } from './types/Todo';
+import { AddInput } from './components/AddInput/AddInput';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Loading } from './components/isLoading';
+import { TodoList } from './components/TodoList';
+import { AddError } from './components/AddError/AddError';
+import {
+  getTodo,
+  deleteTodo,
+  postTodo,
+  updateTodo,
+} from './api/todos';
+import { TodoItem } from './components/TodoItem';
 
-const USER_ID = 0;
+const USER_ID = 7025;
 
 export const App: React.FC = () => {
+  const [todos, setTodo] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.ALL);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisableInput, setIsDisableInput] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const completedTodos = todos.every((todo) => todo.completed);
+
+  const fetchData = async () => {
+    const todosFromServer = await getTodo(USER_ID);
+
+    try {
+      setTodo(todosFromServer);
+      setIsLoading(true);
```

Comment:
Change 'setIsLoading(true)' before you start fetching, because the data has already been fetched.